scripts/seed.js

```diff
@@ -533,11 +570,66 @@ async function createPermissions() {
         object: 'agora',
         scope: 'global',
         description: 'Update the is_on_memberslist status for applications for Agora. Should be assigned to Network Director.'
+    },
+    {
+        action: 'manage_network',
+        object: 'communication_exception',
+        scope: 'global',
+        description: 'Give exceptions to the `communication` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'board_election',
+        scope: 'global',
+        description: 'Set the fulfilment of the `board election` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'members_list',
+        scope: 'global',
+        description: 'Set the fulfilment of the `members list` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'events',
+        scope: 'global',
+        description: 'Set the fulfilment of the `events` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'agora_attendance',
+        scope: 'global',
+        description: 'Set the fulfilment of the `agora attendance` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'development_plan',
+        scope: 'global',
+        description: 'Set the fulfilment of the `development plan` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'fulfilment_report',
+        scope: 'global',
+        description: 'Set the fulfilment of the `fulfilment report` Antenna Criterion'
     }], { individualHooks: true, validate: true });
 
     permissions.networkDirector = [...networkDirectorPermissions, ...permissions.netCom, permissions.seeMemberslistsAgora, permissions.setMemberslistsFeePaidAgora];
 
-    permissions.financialDirector = [permissions.seeMemberslistsAgora, permissions.setMemberslistsFeePaidAgora];
+    const financialDirectorPermissions = await Permission.bulkCreate([{
+        action: 'manage_network',
+        object: 'antenna_criteria',
+        scope: 'global',
+        description: 'Manage the Antenna Criteria fulfilment of Locals'
+    },
+    {
+        action: 'manage_network',
+        object: 'membership_fee',
+        scope: 'global',
+        description: 'Set the fulfilment of the `membership fee` Antenna Criterion'
+    }], { individualHooks: true, validate: true });
```

Comment:
This results in unique constraint errors. You need to create these two separately and handle them similar to `permissions.seeMemberslistsAgora`